relay-event-normalization/src/event.rs

```diff
@@ -977,6 +978,41 @@ pub fn normalize_performance_score(
     version
 }
 
+// Extracts lcp related tags from the trace context
+pub fn normalize_trace_context_tags(event: &mut Event) {
+    let tags = &mut event.tags.value_mut().get_or_insert_with(Tags::default).0;
+    if let Some(contexts) = event.contexts.value() {
+        if let Some(trace_context) = contexts.get::<TraceContext>() {
+            if let Some(data) = trace_context.data.value() {
+                if let Some(lcp_element) = data.lcp_element.value() {
+                    if tags.get("lcp.element").is_none() {
+                        let tag_name = "lcp.element".to_string();
+                        tags.insert(tag_name, Annotated::new(lcp_element.to_string()));
+                    }
+                }
+                if let Some(lcp_size) = data.lcp_size.value() {
+                    if tags.get("lcp.size").is_none() {
+                        let tag_name = "lcp.size".to_string();
+                        tags.insert(tag_name, Annotated::new(lcp_size.try_into().unwrap()));
```

Comment:
We must not unwrap here, that will cause the entire event to be dropped. IMO size does not make sense as a tag because it is a number. If it has to be a tag, I would declare the field on `SpanData` to be of type `LenientString` (that should parse both numbers and strings).

Reply:
👍 Yeah size doesn't really make sense as a tag. This is how we used to send `lcp.size` and the product also expects it this way, so I think we'll need to continue this pattern for now. I've updated to use `LenientString` as you suggested.

Reply:
@jjbayer I had to revisit this because using `LenientString` seemed to have a side effect of mutating the `lcp.size` value in the trace context to a string. I don't think we want this to happen.

I'm not sure if there's a better way to handle this, but I've updated the field in `SpanData` to a `u64` instead. I think this should make sense because the `lcp.size` should always be a positive integer. Let me know if there is something wrong with this!